SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -739,6 +767,59 @@ void FSpatialGDKEditorToolbarModule::StopSpatialServiceButtonClicked()
 	});
 }
 
+void FSpatialGDKEditorToolbarModule::ToggleSpatialDebuggerEditor()
+{
+	// Don't try and start spatial editor debugger spatial networking is disabled.
+	if (!GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
+	{
+		UE_LOG(LogSpatialGDKEditorToolbar, Error, TEXT("Attempted to toggle spatial debugger in editor but spatial networking is disabled."));
```

Comment:
ToggleSpatialDebuggerEditor

Reply:
I will remove this check. I am already disabling the drop-down menu option if multi-worker is not enabled and I will add the spatial networking check there instead. Added manual tests to cover the scenario of spatial networking disabled.